### ISSUE TITLE
Cart-specific tweet threads for gitcoin beta

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#1362]
Because people need to donate separate times for each specific round... we should build tweet threads featuring giveth galaxy projects for each round... and then post a thread of threads w/ all of them.

I started this already with:

OSS --> https://twitter.com/Giveth/status/1652340512479739906

Metacrisis --> https://twitter.com/Giveth/status/1653071549086285832

We should do them also for:
Web3 & Community
Mantle
ENS
Token Engineering

They need basically an intro tweet saying something about the round... and then a link per featured project... explaining it and perhaps explaining how it related to the round. It would be really nice if someone else could take this and draft like 2 before May 2, and 2 before May 3.

Here is my doc with all the drafts so far:
https://docs.google.com/document/d/1n2cMjFW923RvmrLXXZtkBflvKfjV6U4fDjLT1tzR_bs/edit

And here is Suga's blog where the list of projects in each round is (check the cheatsheet at the top). You can get the links from there:
https://blog.giveth.io/17-great-giveth-galaxy-projects-in-the-gitcoin-grants-beta-round-cfa874280a96

@oyealmond @c0ric0ri - can you grab it?

**Comment by oyealmond** [comment]
I can take it @laurenluz :)

**Comment by oyealmond** [comment]
@laurenluz 

Sharing [drafts for community & Mantle tweet threads](https://docs.google.com/document/d/1W7O4Z2ybAAV6LCNr049UATkXIXJ98Ykexszslqs56KU/edit) so you can move forward. 

Working on ENS & Token Engineering next

**Comment by laurenluz** [comment]
Thank you @oyealmond - I tweaked them and scheduled!

ping me again when you got the other two :D

**Comment by oyealmond** [comment]
@laurenluz sharing ENS and TE [tweet threads drafts](https://docs.google.com/document/d/136A42GBbGQbdT7JKNDUfA4BoVNFsRBDF9wVYMkE45Pk/edit).
Let me know your thoughts!

**Comment by laurenluz** [comment]
Hey @oyealmond these are great, they needed a bit of editing work though.

A couple of things I noticed with chat GPT:
- sometimes tweets are overly-wordy/redundant... like the one about the commons stack... it said "governance & funding twice" like the 1st and 2nd setences were kind of repeated - keep an eye on that stuff next time 
- one or two were also above the character limit
- the intro tweets needed a bit of restructuring because they didn't quiiite make sense

I know you also put work into editing things after using AI - adding all the links and tags etc... it's great. just keep an eye on those things next time too.

anyway, they were perfect for this issue, and I edited and scheduled both - and we are good!
